dali/kernels/common/block_setup.h

```diff
@@ -31,6 +31,12 @@ struct BlockDesc {
   ivec<ndim> start, end;
 };
 
+template <>
+struct BlockDesc<1> {
+  int sample_idx;
+  i64vec<1> start, end;
```

Comment:
I believe the this specialization deserves a comment explaining, why we have it

Reply:
done